aidial_assistant/app.py

```diff
@@ -22,7 +21,13 @@
     AssistantApplication,
 )
 
+config_dir = Path(os.getenv("CONFIG_DIR", "aidial_assistant/configs"))
+tools_supporting_deployments: set[str] = set(
+    os.getenv(
+        "TOOLS_SUPPORTING_DEPLOYMENTS", "gpt-4-turbo-1106,anthropic.claude-v2-1"
```

Comment:
Let's make the default an empty string. 

The current default is dependent on **our** **current** core configuration, which is highly contingent, so there is no reason to save this in the repo.

Reply:
Done.